### ISSUE TITLE
Joining as a client causes game to freeze

### ISSUE THREAD

**Issue by Rampant-ai** [Rampant-ai/DRG-Rig-HUD#4]
Report from: https://mod.io/members/handdrawnnerd

Joining a lobby as a client with only this mod enabled causes a game freeze.

- I need to reproduce this, as I have not tested it as a client yet.
- Blueprints have generally been very kind to me when dealing with bad/invalid data, so I'm surprised this results in something so major!

**Comment by Rampant-ai** [comment]
Just by inspecting my blueprints I found a few things that just might go bad in multiplayer. I'll need to give it some explicit testing against a sandbox host, and I will have to reproduce the bug using the `v1.0.0` release to truly verify that I have fixed it.

**Comment by Rampant-ai** [comment]
I was unable to test my fix with a "dev build" as the local mod installation will get disabled even when joining a Sandbox host. However, I uploaded `1.0.1` to mod.io just to test it, and it worked!

Unfortunately, it seems `InitSpacerig` is invoked immediately upon joining the server, so its simple timer delay elapses while most players are still on the "select class" screen. If the `Add to Viewport` function is called at this time, it won't show up properly in the HUD, but if the player clicks through "select class" very quickly (before the timer elapses), it WILL draw correctly. I might open a separate issue for this, since at least I fixed this freeze.
